hops/tls.py

```diff
@@ -215,3 +215,17 @@ def _write_pems():
     client_key_path = os.getcwd() + "/" + constants.SSL_CONFIG.PEM_CLIENT_KEY_CONFIG
     ca_chain_path = os.getcwd() + "/" + constants.SSL_CONFIG.PEM_CA_CHAIN_CERTIFICATE_CONFIG
     _write_pem(k_jks_path, t_jks_path, get_key_store_pwd(), client_certificate_path, client_key_path, ca_chain_path)
+
+def _prepare_rest_appservice_json_request():
+    """
+    Prepares a REST JSON Request to Hopsworks APP-service
+
+    Returns:
+        a dict with keystore cert bytes and password string
+    """
+    key_store_pwd = get_key_store_pwd()
+    key_store_cert = get_key_store_cert()
+    json_contents = {}
+    json_contents[constants.REST_CONFIG.JSON_KEYSTOREPWD] = key_store_pwd
+    json_contents[constants.REST_CONFIG.JSON_KEYSTORE] = key_store_cert.decode("latin-1") # raw bytes is not serializable by JSON -_-
```

Comment:
Is the key_store_cert always stored in "latin-1" collation?

Reply:
Yeah, I think this code will live for at most a couple of weeks. I will open a PR to move all hops-util code to use JWT instead of keystores soon, just need to discuss with Antonis first

Reply:
we will do this for all rest calls if hops-util